NuRadioReco/framework/base_trace.py

```diff
@@ -246,6 +246,75 @@ def deserialize(self, data_pkl):
         if 'trace_start_time' in data.keys():
             self.set_trace_start_time(data['trace_start_time'])
 
+    def add_to_trace(self, channel):
+        """
+        Adds the trace of another channel (with potentially diffrent length and _trace_start_time) to
+        the trace of this channel without changing the sampling. If used on an empty trace with a defined
+        self._sampling_rate and self.self._trace_start_time, and self._time_trace containing zeros, this
+        function can be seen as recording the channel in a specific readout window.
+
+        Parameters
+        ----------
+        channel: BaseTrace
+            Channel to add to current channel
+        """
```

Comment:
In our call last week @sjoerd-bouma was not quite happy about the naming as it can be confused with the `__add__` operator. I am not sure I can come up with a better name... However, I think the doc-string can more clearly reflect this difference. A suggestion from my side (please extend with the information which are missing):
```
        """
        Adds the trace of another channel to the trace of this channel. The traces are only added within the time window of the "this" channel.

        Parameters
        ----------

        channel : BaseTrace
            The channel whose trace is to be added to the trace of this channel.
        """
```

Reply:
Thanks. I think it is clearer now.
One other important difference is that the inputs to __add__ commute whereas they don't for add_to_trace.

Reply:
@MartinRavn I did not commit my proposed doc-string. It was a proposal to you. I agree that can be added to the doc string as well

Reply:
Yes. I have a mix of yours and mine that I will push tomorrow.